pennylane/qchem/tapering.py

```diff
@@ -395,7 +394,20 @@ def taper(h, generators, paulixops, paulix_sector):
     c = anp.multiply(val, h.terms()[0])
     c = qml.math.stack(c)
 
-    return simplify(qml.Hamiltonian(c, o))
+    tapered_ham = simplify(qml.Hamiltonian(c, o))
+    # If simplified Hamiltonian is missing wires, then add wires manually for consistency
```

Comment:
If the simplified Hamiltonian is missing wire, what exactly can go wrong? Is it a problem with tapering the hf state or we know other cases that missing wires, due to simplification, cause problem?

Reply:
It causes problems when tapering non-Hamiltonian observables (number, spin or excitation operators). I noticed that for some molecules, some wires in them go missing upon tapering, which makes them incompatible with the tapered HF state.